test/integration/tasks.test.js

```diff
@@ -328,6 +329,18 @@ describe("Tasks", function () {
 
       return taskId;
     });
+
+    it("Should return 400 if taskid is not exist", async function () {
+      taskId = (await tasks.updateTask(tasksData[5])).taskId;
```

Comment:
what if 1 task is valid and 1 in valid ?

Reply:
Done